pkg/storegateway/indexheader/lazy_binary_reader.go

```diff
@@ -205,6 +205,19 @@ func (r *LazyBinaryReader) LabelNames() ([]string, error) {
 	return r.reader.LabelNames()
 }
 
+// EagerLoad eagerly load index headers from the stored preShutdownLoadedBlocks
```

Comment:
This comment seems out-of-date.

Reply:
Actually the comment almost fine. But lazy_binary_reader.go don't have knowledge about `preShutdownLoadedBlocks`. Hence I updated the godoc to just:

```
// EagerLoad eagerly load index headers.
```